src/services/login.ts

```diff
@@ -1,4 +1,17 @@
-export const signIn = async (email: string, password: string) => {
-  // código
-  console.log(email, password)
+import { signInWithEmailAndPassword, Auth } from 'firebase/auth'
+
+export const signInService = async (auth: Auth, email: string, password: string) => {
+  try {
+    await signInWithEmailAndPassword(auth, email, password)
+    if (auth.currentUser) {
+      return { success: true }
+    } else {
+      return { success: false, message: 'User is not logged in' }
```

Comment:
se puede destructurar el user, luego se puede retornar el objeto para saber que fue todo bien
```suggestion
    const { user } = await signInWithEmailAndPassword(auth, email, password)
    return { success: true, user }
```

Reply:
Buena observacion! gracias!